samples/wifi/wfa_qt_app/README.rst

```diff
@@ -56,10 +95,62 @@ Also, run the following command to create a Serial Line Internet Protocol (SLIP)
+   tunslip6 -s <serial_port> -T <IPv6_prefix>
+
+* `tunslip6`: tunslip6 creates a SLIP interface on the host PC. which can be used for serial communication.
+* `serial port`: This can be replaced with device path on which DUT is connected.
+* `IPv6_prefix`: This can be replaced with the desired IPv6 address and subnet prefix length for your device.
+
+Example command:
+
+.. code-block:: console
+
+   tunslip6 -s /dev/ttyACM4 -T 2001:db8::1/64
+
+.. note::
+   This can be done only on Linux PC.
 
 See also :ref:`cmake_options` for instructions on how to provide CMake options.
 
+Test execution
+==============
+
+After programming, the sample shows the following output:
+
+.. code-block:: console
+
+   *** Booting nRF Connect SDK 48f33c9870f1 ***
+   Starting nrf7002dk_nrf5340_cpuapp with CPU frequency: 128 MHz
+   [00:00:00.330,932] <inf> net_config: Initializing network
+   [00:00:00.330,932] <inf> net_config: Waiting interface 2 (0x20007a58) to be up...
+   [00:00:00.331,085] <inf> wpa_supp: Successfully initialized wpa_supplicant
+   [00:00:00.331,237] <inf> wfa_qt: Welcome to use QuickTrack Control App DUT version v2.1
+
+   [00:00:00.331,268] <inf> wfa_qt: QuickTrack control app running at: 9004
+   [00:00:00.331,268] <inf> wfa_qt: Wireless Interface: wlan0
+   [00:00:00.757,965] <inf> net_config: Interface 1 (0x20007940) coming up
+   [00:00:00.758,087] <inf> net_config: Running dhcpv4 client...
+   [00:00:00.763,732] <inf> usb_ecm: Set Interface 0 Packet Filter 0x000c not supported
+   [00:00:00.830,627] <inf> usb_ecm: Set Interface 0 Packet Filter 0x000e not supported
+   [00:00:00.832,214] <inf> usb_ecm: Set Interface 0 Packet Filter 0x000e not supported
+   [00:00:00.832,519] <inf> usb_ecm: Set Interface 0 Packet Filter 0x000e not supported
+   [00:00:00.834,594] <inf> usb_ecm: Set Interface 0 Packet Filter 0x000e not supported
+   [00:00:00.835,021] <inf> usb_ecm: Set Interface 0 Packet Filter 0x000e not supported
+   [00:00:00.835,327] <inf> usb_ecm: Set Interface 0 Packet Filter 0x000e not supported
+   [00:00:00.835,937] <inf> usb_ecm: Set Interface 0 Packet Filter 0x000e not supported
+
+Quicktrack certification testing
+********************************
+
+* QuickTrack certification tests can be done by individual companies with Wi-Fi Alliance membership or by authorised test houses.
+* See :ref: `https://www.wi-fi.org/system/files/members/QuickTrack Test Tool Platform Integration Guide v1.0.pdf` for the installation process of relevant access points.
+
+.. note::
+   OpenWRT access points have known limitations running STAUT Security Vulnerability Detection test cases. Wi-Fi Alliance recommends use of Intel AX210 station as SoftAP to perform these tests.
+
+See :ref: `https://www.wi-fi.org/file-member/quicktrack-test-tool-platform-intel-ax210-station-integration-guide` for more details.
+
+* See :ref: `https://www.wi-fi.org/system/files/members/QuickTrack Test Tool User Manual v2.2.pdf` for installing QuickTrack test tool.
```

Comment:
Can we add line 153 somewhere higher in the documentation?

Reply:
Yeah, added just 2 lines above.